datadog/resource_datadog_synthetics_test_.go

```diff
@@ -126,6 +127,35 @@ func resourceDatadogSyntheticsTest() *schema.Resource {
 					},
 				},
 			},
+			"variables": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"example": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"id": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"name": {
```

Comment:
The format of a variable's name needs to follow a pattern, exactly the same as this one: https://github.com/DataDog/terraform-provider-datadog/blob/master/datadog/resource_datadog_synthetics_global_variable.go#L24